src/index.js

```diff
@@ -1,7 +1,6 @@
 "use strict";
 
 var domino = require("domino");
-var validateElementName = require("validate-element-name");
```

Comment:
Just spotted this - we should remove the dependency if we're not using this any more.